impress-org/givewp
# Feature: Add Recurring add-on product overlay to Reports widget

# Description

This PR implements a modal-like overlay on the Reports Widget in the admin dashboard. Its purpose is to provide users with information about the recurring add-on if they are unfamiliar with it. The dialog box also includes a button that opens a new window and directs users to a page where they can read about and purchase the Recurring add on. The overlay and dialog will only appear to users who have not received any donations yet.

## Affects

Wordpress Admin page Reports Widget

## Visuals

https://github.com/impress-org/givewp/assets/75056371/20efce11-4441-43b3-9967-f51b8bebba04


## Testing Instructions

- Ensure you have no donations.
- Visit the Admin dashboard and view the Reports Widget. 
- Verify the dialog box is displayed and verify the UTM link is correct and opens Recurring add-on page in a new window.
- Make a donation.
- Revisit the widget and verify the dialog is no longer displayed.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204875481837188
